env.sample

```diff
@@ -2,7 +2,7 @@
 # your own before running run.py.
 
 # Watson Assistant
-WORKSPACE_ID=<add_assistant_workspace>
```

Comment:
don't you still need this? 
env_workspace_id = environ.get('WORKSPACE_ID')

Reply:
yes in this case. But I will rename it to `ASSISTANT_ID`

Reply:
why rename it?  We've used WORKSPACE_ID here and other places for years.  Is ASSISTANT_ID a thing used other places?  I understand it might need a rename w/ the latest, but this looks odd.

Reply:
Noticed in the review that it is referred to as "SKILL ID" so that would make sense.  Still not sure ASSISTANT_ID makes sense.  Does it match something in the Assistant UI? or well-known IDs.

Reply:
Just looked at the UI.  "SKILL_ID" is certainly a good update since workspace term isn't used anymore. Also I agree it'd be nice to have an "ASSISTANT_" prefix to kind of group it with the official ones.  I'd vote for "ASSISTANT_SKILL_ID" if I was renaming it.

Reply:
The change to "Skill ID" in the UI is confusing since the API still uses the term workspaceID (node) and workspace_id (python).